### PR TITLE
Fix for boost.python issue ( thx Juan )

### PR DESCRIPTION
**Pull request by adevress**
- Solve issue with undetected boost.python when trying to compile with python 2.7.X
- Update the nix build recipe

### CODE REVIEW

**Comment by adevress** [comment]
@hernando ping :)

File: default.nix [hunk 1]
```diff
@@ -11,19 +11,20 @@
...
 with import <BBPpkgs> { };
 
+{
 
-stdenv.mkDerivation rec {
+brion = brion.overrideDerivation (oldAttr: rec {
       name = "Brion-DEV";
       src = ./.;
-      buildInputs = [stdenv pkgconfig boost zlib cmake cmake-external hdf5-cpp servus lunchbox vmmlib mvdtool bbptestdata ];
-
```
Comment (hernando) [inline default.nix]: I will simply believe you in this one.

**Comment by adevress** [comment]
Thx @hernando 

Does somebody can force the pull request ? The OSX build with travis is incomplete and do not  include boost.

I need this merged to deploy RT neuron at Jureca and Cineca

**Comment by eile** [comment]
Well, your change seems to break finding boost on OS X... So I don't think it should be merged.

**Comment by hernando** [comment]
I didn't merge it yesterday for that reason. Can you push to a boost_python branch in this repo so I can amend it?

**Comment by adevress** [comment]
@hernando  I am not allowed to pushed to this repi for now, I need someone with right to allow me

@eile  Well, your travis recipe do not install boost, you are depending of the little subset of boost installed by default....

**Comment by adevress** [comment]
new commit with fix from #96

**Pull request merged**